src/common/models/filter/filter.ts

```diff
@@ -119,6 +119,15 @@ export class Filter implements Instance<FilterValue, FilterJS> {
     }
   }
 
+  public isDynamic(): boolean {
+    return this.clauses.some(clause => clause.dynamic);
+  }
+
```

Comment:
is it important for callers of this to know that some processing is happening? (vs named getFilter)